src/console/migrations/m170705_095702_webform.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use yii\db\Migration;
+
+class m170705_095702_webform extends Migration
+{
+    public function safeUp()
```

Comment:
@miguelbemartin can you remove the safeUp and safeDown methods that aren't being used.